__tests__/app.test.js

```diff
@@ -350,4 +350,32 @@ describe.only("DELETE /api/comments/:comment_id", () => {
         expect(msg).toBe('not found');
       });
   });
-});
\ No newline at end of file
+});
+
+describe("GET /api/users", () => {
+  test("200: Responds with an array of user objects", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body: { users } }) => {
+        expect(Array.isArray(users)).toBe(true);
+        expect(users.length).not.toBe(0);
+        users.forEach((user) => {
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String)
+          });
+        });
+      });
+  });
+
+  test("404: Responds with a message of 'Bad Request' when the URL is not found", () => {
+    return request(app)
+      .get("/api/usesr")
```

Comment:
This isn't unique to this endpoint. If the user types anything incorrect after /api/ it's something that could happen at any point and is a general server error. It can therefore live in it's own describe block 👍

Reply:
I've actually have that test in a separate describe block. I just felt like it's necessary to do so I put it in my test suite and wait the tutor to comment. :)

Regardless, I've deleted it.